src/rtop/jbuild

```diff
@@ -2,6 +2,7 @@
 
 (install
  ((section bin)
+  (package reason)
```

Comment:
Was this needed? Why?

Reply:
This is needed because when you have multiple packages in one opam repository, jbuilder needs to know which binary belongs to which package.